india_compliance/gst_india/report/summary_of_inward_supplies/summary_of_inward_supplies.py

```
@@ -0,0 +1,317 @@
+
+        elif gst_category != "Unregistered" and is_reverse_charge:
+            return InwardSuppliesCategory.REG_RCM
+
+        elif itc_classification == "Import Of Goods":
+            return InwardSuppliesCategory.IMPORT_GOODS
+
+        elif itc_classification == "Import Of Service":
+            return InwardSuppliesCategory.IMPORT_SERVICES
+
+        elif itc_classification == "Input Service Distributor":
+            return InwardSuppliesCategory.ITC_FROM_ISD
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)